automation/test.sh

```diff
@@ -26,6 +26,7 @@ trap cleanup EXIT
 test_id=$(docker run --privileged -d \
 	--tmpfs /run \
 	--tmpfs /sys/fs/cgroup \
+	-v "${PWD}":/usr/src/app \
```

Comment:
This looks like a leftover from local testing?

Reply:
It was needed because of how the tests are run: We build the open-balena-vpn container, then run a custom set of commands inside it before invoke the tests. The problem being that the open-balena-vpn container no longer contains the files necessary to run the tests. As far as I could see this left me with two options. Either bind-mount the source folder as above, or make use of `docker cp` to copy the test files into the container.

Perhaps the latter, being more explicit, would be better?